source/retryable-reads/tests/count-serverErrors.yml

```diff
@@ -0,0 +1,141 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "Count succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [count], errorCode: 11600 }
+        operation: &retryable_operation
+            name: count
```

Comment:
Should this be tested even though the helper is deprecated? Most drivers still have the method so I suppose there's good reason to do so.

I'm curious if drivers like Go and Swift, which will have their first public release after the deprecation, will need special logic to skip this. I suppose it's not a huge problem provided all of the `count()` helper tests are in their own file (they can just omit the test files in their runners).

Reply:
I concur that it's easy to skip if needed, since all `count()` tests are in their own file(s). LGTM.